### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by artemnovichkov**
Add Makefile and scripts to simplify steps during initializing and bootstrapping:
- Check required dependencies
- Using Fastfood to add last fastlane lanes from RSBFastlane
- Uncomment correct flags from fastlane/.env

### CODE REVIEW

**Review by stason314 — COMMENTED** [review]
Test

File: hooks/post_gen_project.sh [hunk 1]
```diff
@@ -3,8 +3,7 @@
 set -e
 
 {% if cookiecutter.fabric == "Yes" %}
-carthage update --platform iOS
-echo "✅ Fabric frameworks were added successfully. Don't forget to add API Key to Info.plist and API Key and Build Secret to Fabric Run Script Build Phase. The credentials are available here: https://www.fabric.io/settings/organizations"
+echo "⚠️ Don't forget to add API Key to Info.plist and API Key and Build Secret to Fabric Run Script Build Phase. The credentials are available here: https://www.fabric.io/settings/organizations"
```
Comment (ntunin) [inline hooks/post_gen_project.sh:6]: New projects will not presented in Fabric Beta

File: {{ cookiecutter.name }}/README.md [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
 	```bash
-	brew install carthage
-	```
-2. Run a command in a project directory:
-
-	```bash
-	carthage bootstrap --platform iOS
+	make bootstrap
 	```
 3. Open `{{ cookiecutter.name }}.xcodeproj` in last Xcode version.
```
Comment (ntunin) [inline {{ cookiecutter.name }}/README.md]: What about 2.?

**Review by ntunin — CHANGES_REQUESTED** [review]

**Review by stason314 — APPROVED** [review]

File: hooks/post_gen_project.sh (hunk 1 above)
Comment (artemnovichkov) [reply hooks/post_gen_project.sh:6]: Thanks for the comment! I've added a task to command backlog.

**Pull request merged**